packages/photon/src/decs.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'crc32' {
+	export default function crc32( s: string ): string;
```

Comment:
Unfortunately no type package for this one -- looks like it hasn't been updated in 10 years. :eyes: